perseo-main/src/test/java/com/telefonica/iot/perseo/test/Help.java

```diff
@@ -55,6 +55,8 @@
                  ,"expression E {(v1,v2) => max(v1,v2)} select E(1, 2) from iotEvent"
                  ,"expression E {(v) => case when v is null or cast(v,float) < 0 or cast(v,string) = 'null' or cast(v,string) = ' ' then 0 else v end}  select E(5) from iotEvent"
                  ,"select current_timestamp() as r from iotEvent where (cast(price?,String) != '321' and (cast(price?,String) != '123'))"
+                 ,"insert into mynewstream select window(price) as winprice from iotEvent.win:length(2) as price"
+                 ,"select *, winprice.get(0) as primero from mynewstream having count(winprice)>1"
```

Comment:
CNR entry missing?

Maybe not, if this is just an informative message...

Reply:
Btw, "primero" is a typo? Should it be "first"?

Reply:
No CNR, is just an example